install/0000_80_machine-config-operator_07_etcdquorumguard_deployment.yaml

```diff
@@ -49,16 +50,23 @@ spec:
         operator: Exists
         effect: NoSchedule
       containers:
-      - image: registry.svc.ci.openshift.org/openshift:cli
+      - image: quay.io/openshift/origin-cli:latest
         imagePullPolicy: IfNotPresent
         name: guard
         volumeMounts:
         - mountPath: /mnt/kube
           name: kubecerts
         command:
-        - "/bin/sleep"
+        - /bin/bash
         args:
-        - "infinity"
+        - -c
+        - |
+          # properly handle TERM and exit as soon as it is signaled
+          set -euo pipefail
+          trap 'jobs -p | xargs -r kill; exit 0' TERM
+          while true; do
```

Comment:
Why this as opposed to

`sleep infinity & wait`

or simply 

`sleep infinity`?

Reply:
that's fine